reana_server/status.py

```diff
@@ -602,33 +604,40 @@ def __init__(self):
     def get_health_status(percentage):
         """Calculate quota health status."""
         health = ClusterHealthStatus.healthy
-        if percentage >= 50:
-            if percentage >= 75:
-                health = ClusterHealthStatus.critical
-            else:
-                health = ClusterHealthStatus.warning
+        if percentage <= 50:
+            health = ClusterHealthStatus.warning
+        if percentage <= 25:
+            health = ClusterHealthStatus.critical
+
         return health.name
 
     @staticmethod
-    def get_percentage(amount, total):
-        """Get percentage value based on ``amount`` and ``total``."""
-        return round((amount / total if total else 0) * 100)
+    def get_percentage(used, total):
+        """Get usage percentage based on ``used`` and ``total``."""
+        percentage = 100 - round((used / total if total else 0) * 100)
+        return percentage if percentage > 0 else 0
+
+    @staticmethod
+    def get_available(used, total):
+        """Get available cluster resources based on ``used`` and ``total``."""
+        available = total - used
+        return available if available > 0 else 0
 
     def get_node_health(self):
         """Get cluster nodes health information."""
         nodes_status_obj = NodesStatus()
 
         total = len(nodes_status_obj.get_nodes())
         unschedulable = len(nodes_status_obj.get_unschedulable_nodes())
+        available = ClusterHealth.get_available(unschedulable, total)
```

Comment:
This might be a bit confusing as we're passing `unschedulable` as `used` arg, which I don't think is the same concept. In fact, I'm not sure if it's possible to get the `used` nodes.

Reply:
Indeed this chart is a bit different, it shows live nodes instead of used ones. I guess it should be a bit problematic to show how busy are the nodes in a current moment.. That's why we use `unschedulable` as `used` arg. Are you proposing to change `used` arg name in `get_available()`?

Reply:
> Are you proposing to change `used` arg name in `get_available()`?

Yes, that was my idea.. Making the function a little bit more generic perhaps.

Reply:
Change to `unavailable` instead of `used` and it covers both `used` and `unschedulable`.